bioengine_apps/cellpose_finetuning/main.py

```diff
@@ -849,13 +929,18 @@ def train_seg_with_callbacks(
                 batch_idx = k // batch_size
                 total_batches = (nimg_per_epoch + batch_size - 1) // batch_size
                 batch_loss_per_sample = loss.item()  # Loss per sample for this batch
-                batch_callback(iepoch + 1, batch_idx, total_batches, batch_loss_per_sample, elapsed)
+                batch_callback(
```

Comment:
I am not sure, but would it make sense to let batch_callback also carry another argument for the test metrics? and can be None for training but with value for validation? does that make sense? I am not sure though.